osd_whats_new/osd-whats-new.adoc

```diff
@@ -62,3 +62,7 @@ For more information on region availabilities, see xref:../osd_architecture/osd_
 * **Cluster install update.** {product-title} clusters can now be installed on {GCP} shared VPCs. For more information, see xref:../osd_install_access_delete_cluster/creating-a-gcp-cluster.adoc#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp[Creating a cluster on GCP with CCS].
 
 * **{product-title} on Google Cloud Marketplace availability.** When creating an {product-title} (OSD) cluster on Google Cloud through the Hybrid Cloud Console, customers can now select Google Cloud Marketplace as their preferred billing model. This billing model allows Red Hat customers to take advantage of their link:https://cloud.google.com/docs/cuds[Google Committed Use Discounts (CUD)] towards {product-title} purchased through the Google Cloud Marketplace. For more information, see xref:../osd_install_access_delete_cluster/creating-a-gcp-cluster.adoc#osd-create-cluster-gcp-account_osd-creating-a-cluster-on-gcp[Creating a cluster on GCP with Google Cloud Marketplace].
+
+[id="osd-known-issues_{context}"]
+== Known issues
+* {OCP} 4.14 introduces an updated HAProxy image from 2.2 to 2.6. This update introduces a change in behavior enforcing strict RFC 7230 compliance, rejecting requests with multiple `Transfer-Encoding` headers. This may cause exposed pods in {product-title} 4.14 clusters sending multiple `Transfer-Encoding` headers to respond with a `502 Bad Gateway` or `400 Bad Request error`. To avoid this issue, ensure that your applications are not sending multiple `Transfer-Encoding` headers. For more information, see link:https://access.redhat.com/solutions/7055002[Red Hat Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-43095[*OCPBUGS-43095*])
```

Comment:
I see that you added "To avoid this issue..." and comma we discussed. Thanks!

Reply:
done thanks